### PR TITLE
COP-233 Model nationality + country schemas in ref-data-api

### PR DESCRIPTION
**Pull request by ricca509**
Schemas:

- https://github.com/UKHomeOffice/RefData/blob/master/schemas/reference/V12__country.sql
- https://github.com/UKHomeOffice/RefData/blob/master/schemas/reference/V13__nationality.sql

### CODE REVIEW

File: app/db/models/country.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      comment:
+        '{"label": "Linked Country", "description": "If the state is not recognised by the Foreign and Commonwealth Office as Sovereign, then this Country handles its International Relations.", "summaryview": "TRUE"}',
+    },
+    relationshipwithlinkedcountry: {
+      type: DataTypes.STRING(90),
+      comment:
+        '{"label": "Relationship with linked Country", "description": "Relationship with another country, if there is one.", "summaryview": "TRUE"}',
+    },
+    fullname: {
+      type: DataTypes.STRING(90),
```
Comment (joshuadiablito) [inline app/db/models/country.js:77]: https://github.com/UKHomeOffice/RefData/blob/master/schemas/reference/V12__country.sql#L15 fullname is currently 80? probably wont create any issues by making it more I am sure...

File: app/db/models/country.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+      comment:
+        '{"label": "Commonwealth membership", "description": "Is this country/territory/geographic area a member of the Commonwealth.", "summaryview": "false"}',
+    },
+    eu: {
+      type: DataTypes.BOOLEAN,
+      allowNull: false,
+      comment:
+        '{"label": "EU membership", "description": "Is this country/territory/geographic area a member of the European Union.", "summaryview": "false"}',
+    },
+    eaa: {
```
Comment (joshuadiablito) [inline app/db/models/country.js:115]: `eea` not `eaa` https://github.com/UKHomeOffice/RefData/blob/master/schemas/reference/V12__country.sql#L22

File: app/db/models/country.js (hunk 1 above)
Comment (ricca509) [reply app/db/models/country.js]: Done, thanks!

File: app/db/models/country.js (hunk 2 above)
Comment (ricca509) [reply app/db/models/country.js]: 👍

**Pull request merged**